Makefile

```diff
@@ -6,45 +6,78 @@ build:
 test:
 	./gradlew test
 
+clean:
+	rm -rf ./build-artifacts/*
+	rm -rf ./smoke-tests/apps/*.jar
+	rm -rf ./smoke-tests/collector/data.json
+	rm -rf ./smoke-tests/report.*
+	./gradlew clean
+
 project_version:=$(shell grep 'project.version =' build.gradle | awk -F\" '{ print $$2 }')
 #: display the project's version
 project_version:
 	@echo ${project_version}
 
-dc-agent-only=docker-compose --file ./smoke-tests/smoke-tests-agent-only/docker-compose.yml
-dc-agent-manual=docker-compose --file ./smoke-tests/smoke-tests-agent-manual/docker-compose.yml
+build-artifacts:
+	mkdir -p ./build-artifacts
+
+smoke-tests/collector/data.json:
+	@echo ""
+	@echo "+++ Zhuzhing smoke test's Collector data.json"
+	@touch $@ && chmod o+w $@
```

Comment:
This is a fix for permissions errors in0 the collector container when this is the file mounted in for the fileexporter.